contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -521,6 +529,21 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
         );
     }
 
+    /**
+     * Retrieves the length of the queue.
+     * @return Length of the queue.
+     */
+    function _getQueueLength(
+    )
+        internal
+        view
+        returns (
+            uint40
+        )
+    {
+        return queue.getLength()/2;
```

Comment:
We have to divide by 2 because we store 2 values for every queue element

Reply:
You should add this comment to the source code